File: go_servers/go_servers_info/models/domain.go
package models

// Domain : struct
type Domain struct {
	ID              int64    `sql:",pk,unique,notnull"`
	Servers         []Server `json:"servers"`
	ServerChanged   bool     `json:"servers_changed"`
	SslGrade        string   `json:"ssl_grade"`
	PreviusSslGrade string   `json:"previus_ssl_grade"`
	Logo            string   `json:"logo"`
	Title           string   `json:"title"`
	IsDown          bool     `json:"is_down"`
}

File: go_servers/go_servers_app/go_servers_app/Models/Domain.swift
import Foundation
import EVReflection

class Domain: EVObject {
    var ID: Int64? = 0
    var servers: [Server]? = []
    var serverChanged: Bool? = false
    var sslGrade: String? = ""
    var previusSslGrade: String? = ""
    var logo: String? = ""
    var title: String? = ""
    var isDown: Bool? = false
}
